spec/lib/availability_status_listener_spec.rb

```diff
@@ -101,4 +104,21 @@
       end
     end
   end
+
+  context "when kafka at localhost is not available" do
```

Comment:
actually it's testing only the "I don't care" part from production's (stage/qa/ci) point of view